content/web-frameworks/gatsby/why-gatsby.md

```diff
@@ -31,6 +35,17 @@ showToc: true
   - [GitHub Action workflow for GitHub Pages](/programming-cookbook/github-actions/deploy-to-github-pages)
   - [GitHub Action workflow for CloudFlare Workers](/programming-cookbook/github-actions/deploy-to-cloudflare-workers)
 
+# Gatsby don't be suitable if:
```

Comment:
Very minor, suitable is adjective. Then we use it with `is`.

`Gatsby isn't suitable if:`

Reply:
Oh, it's my fault. Thank a lot.

Reply:
Did you fix this?

Reply:
Yes, thanks